portal/portal-api/api/src/java/org/sakaiproject/portal/api/PortalService.java

```diff
@@ -46,32 +47,32 @@ public interface PortalService
 	 * current request. It should be a string, and should be implimented where
 	 * the request is portlet dispatched.
 	 */
-	public static final String PLACEMENT_ATTRIBUTE = PortalService.class.getName() + "_placementid";
+    String PLACEMENT_ATTRIBUTE = PortalService.class.getName() + "_placementid";
```

Comment:
Indentation here, and .. should these not be kept as final, just static final instead of public? Upper case implies a constant.

Reply:
final is redundant for interface fields